charts/airflow/README.md

```diff
@@ -214,6 +214,16 @@ airflow:
           key: password
 ```
 
+You can also use `airflow.extraEnvFrom` to mount all the values in a ConfigMap or Secret
+For example, mounting all the data in `custom-config-map` and `super-secrets` as environment variables (the `custom-config-map` and `super-secrets` must already exist):
+```yaml
```

Comment:
I think this text should be:
```md
We expose the `airflow.extraEnvFrom` value to mount all keys inside a ConfigMap or Secret as environment variables.

For example, mounting keys from `ConfigMap/my-airflow-configs` and `Secret/my-airflow-secrets`:
```yaml
airflow:
  extraEnvFrom:
    - configMapRef:
        name: my-airflow-configs
    - secretRef:
        name: my-airflow-secrets